xtc/src/fee.rs

```diff
@@ -0,0 +1,9 @@
+/// Compute the fee that should be taken for a transaction based on the total amount
```

Comment:
I think the comment above kind of contradicts the implementation which doesn't take the amount into account :)

I think it is a good idea to use a fixed amount as you did, but maybe we can improve on it.
We could do something like this:
fee = min(max(2_000_000_000, amount * X%), 10_000_000_000)

so if someone transfers a larger amount that he pays more, but there is a cap of 10_000_000_000. This cap would encourage people who want to transfer let's say 10^16 cycles to use only 1 transaction instead of breaking it up to multiple smaller transactions.

regarding floating point calculations(* X%), web assembly is using an IEEE standard, so regardless of the architecture of the IC nodes, the result is the same on all nodes
https://webassembly.github.io/spec/core/syntax/types.html